### PR TITLE
PR: Updated to V2 of Clarifai API

### PR DESCRIPTION
**Pull request by ModerateContent**
## Updated this plugin to get it working again.

**Changes**
- now works with Clarifai API V2
- requires the user to enter new Clarifai API key 

**Tested on**
- PC Windows 10 Lightroom 6
- Mac OSX Lightroom 6

### CODE REVIEW

File: ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua [hunk 1]
```diff
@@ -1,68 +1,56 @@
...
 local JSON = require 'JSON'
 local prefs = import 'LrPrefs'.prefsForPlugin(_PLUGIN.id)
 
 local logger = LrLogger('ClarifaiAPI')
 logger:enable('print')
 
 
-local tagAPIURL   = 'https://api.clarifai.com/v1/tag/'
-local tokenAPIURL = 'https://api.clarifai.com/v1/token/'
+local tagAPIURL   = 'https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs'
```
Comment (SeidChr) [inline ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua:13]: this id in the url doesnt look right. sure it belongs here?

Comment (ModerateContent) [reply ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua:13]: The id in the URL is the image recognition model ID for Carifai's "general" recognition model and not any sort of authentication id. See the model ID at this link

https://clarifai.com/models/general-image-recognition-model/aaa03c23b3724a16a56b629203edc62c

I'm guessing that's what you're asking.

Comment (SeidChr) [reply ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua:13]: Yes, great. Thank you!
Now someone has to merge it :)
Will use your revision in the meantime.

Comment (SeidChr) [reply ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua:13]: Tested it. Working great! (no crashes ;))

**Pull request merged**